admin_manual/installation/nginx_configuration.rst

```diff
@@ -80,8 +80,10 @@ Nginx Configuration
       rewrite ^/.well-known/host-meta /public.php?service=host-meta last;
       rewrite ^/.well-known/host-meta.json /public.php?service=host-meta-json last;
```

Comment:
Those two seems to causing a 404 / Not Found here and also on the demo.owncloud.org page:

```
curl -i https://demo.owncloud.org/.well-known/host-meta

HTTP/1.1 404 Not Found
Server: nginx
Date: Tue, 13 Oct 2015 12:36:42 GMT
Content-Type: text/html; charset=UTF-8
Content-Length: 0
Connection: keep-alive
Set-Cookie: 52400eaba833a=n8ce4tpqm2877082fcp1u5r2i5; path=/; HttpOnly
Expires: Thu, 19 Nov 1981 08:52:00 GMT
Cache-Control: no-store, no-cache, must-revalidate, post-check=0, pre-check=0
Pragma: no-cache
Content-Security-Policy: default-src 'self'; script-src stats.owncloud.org 'self' 'unsafe-eval' 'unsafe-inline'; style-src 'self' 'unsafe-inline'; frame-src *; img-src *; font-src 'self' data:; media-src *; connect-src *
X-XSS-Protection: 1; mode=block
X-Content-Type-Options: nosniff
X-Frame-Options: Sameorigin
X-Robots-Tag: none
Accept-Ranges: bytes
Age: 0
Via: seetheprogress caching
X-Powered-By: community guy
X-Paid: No
```